cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -636,7 +636,7 @@ constructTxBodyContent
   -- ^ Withdrawals
   -> [Hash PaymentKey]
   -- ^ Required signers
-  -> Maybe L.Coin
+  -> L.Coin
```

Comment:
I'm unsure about this bit. I guess it makes sense to say in the function signature, that the fee is required and let users specify 0 if needed.